### DISCUSSION TITLE
GitHub Actions status badge always show failing with path URL

### DISCUSSION THREAD

**Discussion by peaceiris** [community/community#27098] [Actions]
<p>For example, in this repository, <a href="https://github.com/peaceiris/actions-gh-pages" title="GitHub Actions for GitHub Pages" rel="nofollow noopener">GitHub Actions for GitHub Pages</a>, I put the workflow status badge with workflow name type URL.</p>
<p><a href="https://github.com/peaceiris/actions-gh-pages/workflows/docker-image-ci/badge.svg" rel="nofollow noopener">https://github.com/peaceiris/actions-gh-pages/workflows/docker-image-ci/badge.svg</a></p>
<p>That shows “passing” and is expected behavior.</p>
<p>Also, the path type badges are available like the following:</p>
<p><a href="https://github.com/peaceiris/actions-gh-pages/workflows/.github/workflows/docker-image-ci.yml/badge.svg" rel="nofollow noopener">https://github.com/peaceiris/actions-gh-pages/workflows/.github/workflows/docker-image-ci.yml/badge.svg</a></p>
<p>It always shows “failing”. That seems to be a bug.</p>

**Comment by weide-zhou** [comment]
<p>Hi,</p>
<p>Thanks for your feedback! </p>
<p>Please remove ‘name’ for your workflow, I copied your repo and the badge for path can display well on my side.</p>
<p>If the name keyword exists for your workflow, please just use workflow name for the badge.</p>
<p>Related link for more info, thanks.</p>
<p><a href="https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fhelp.github.com%2Fen%2Factions%2Fautomating-your-workflow-with-github-actions%2Fconfiguring-a-workflow%23adding-a-workflow-status-badge-to-your-repository&amp;data=02%7C01%7Cv-wadzho%40microsoft.com%7C96977e4c40824d719e2a08d76cba2730%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637097419083607476&amp;sdata=1kfZJ4Pr%2BFpMY2lm0cDX359Ot%2F0BjIO7F8KBLNyguvg%3D&amp;reserved=0" rel="nofollow noopener">https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow#adding-a-workflow-status-badge-to-your-repository</a></p>
<p><img src="https://user-images.githubusercontent.com/52772761/181101686-832c8bdf-b5f5-435e-8ea8-23a8ebc3162e.png" alt="BadgewithFilePath.png" data-base62-sha1="bOpoE7TOlRkKTVtz6oBEBIr5pJ2" width="576" height="266"></p>

**Comment by IgorGanapolsky** [comment]
<p>The related link you posted doesn’t show info about badges…</p>

**Comment by peaceiris** [comment, marked as answer]
<p>The latest link is here: <a href="https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-the-workflow-file-name" class="inline-onebox" rel="noopener nofollow ugc">Adding a workflow status badge - GitHub Docs</a></p>
